components/request_otr/common/features.cc

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/request_otr/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace request_otr {
+namespace features {
+
+// When enabled, Brave will block domains listed in the Brave-maintained
+// request-OTR ('off-the-record') list, or any server that responds with a
+// 'X-Request-OTR: 1' HTTP header, and present a security interstitial with
+// choice to proceed normally or in an off-the-record context.
+BASE_FEATURE(kBraveRequestOTR,
```

Comment:
I thought we were going with RequestOTRTab? https://bravesoftware.slack.com/archives/G011DCVPKE3/p1681512144168739

Reply:
also the naming should be consistent everywhere so class names should be `RequestOTRTab*`

Reply:
as per discussion in Slack, renaming the feature flag only, for now